src/Components/Auth/ForgotPassword.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { TextField, Button, Typography, Box } from "@mui/material";
+
+const ForgotPassword = () => {
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const res = await fetch(
+        `${import.meta.env.VITE_API_BASE_URL}/api/v1/auth/request-reset-password`,
+        {
+          method: "POST",
+          headers: { "Content-Type": "application/json" },
+          body: JSON.stringify({ email }),
+        },
+      );
+
+      if (res.ok) {
+        setMessage("Password reset email sent! Check your inbox.");
+      } else {
+        const data = await res.json();
+        setMessage(data.msg || "Something went wrong.");
+      }
```

Comment:
I have an error in the console but UI doesn't show anything: 
```
Error Handler Middleware false CustomAPIError: User with this email doesn't exist.
    at requestPasswordReset (/Users/nihalerdal/Desktop/Practicum2/ii-practicum-team-5-back/src/controllers/auth.ts:81:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) {
  statusCode: 404
}
```
That'd be great if we could add alert messages for those error coming from backend. You can use Alert component of MUI. For example: 
 ```
{/* Error Message */}
              {errorMessage && (
                <Alert severity="error" sx={{ mt: 2 }}>
                  {errorMessage}
                </Alert>
              )}
```